### DISCUSSION TITLE
How to install playebots?

### DISCUSSION THREAD

**Discussion by deadcorpseeater** [azerothcore/azerothcore-wotlk#15412] [Modules]
I can't get what's a branch author is talking about? HOw to compile srver to make it work? I see some files in his repository looks like azerothcore ones

**Comment by Nyeriah** [comment]
Follow the instructions written in the module: https://github.com/ZhengPeiRu21/mod-playerbots

You must use **Zheng Pei Ru's** fork of Azerothcore. The link is provided in the instructions.

**Reply by deadcorpseeater** [reply]
what is branch? how to use it? there'is no step-by-step wxplanation

**Reply by Nyeriah** [reply]
The link provided in the instalation guide leads to the Playerbot branch.

https://github.com/ZhengPeiRu21/azerothcore-wotlk/tree/Playerbot

![image](https://user-images.githubusercontent.com/47818697/225694963-483562dc-ef92-4bf8-a111-92b3cc42a500.png)

From there it's just a normal installation like every other module, I presume.

**Reply by deadcorpseeater** [reply]
> The link provided in the instalation guide leads to the Playerbot branch.
> 
> https://github.com/ZhengPeiRu21/azerothcore-wotlk/tree/Playerbot
> 
> ![image](https://user-images.githubusercontent.com/47818697/225694963-483562dc-ef92-4bf8-a111-92b3cc42a500.png)
> 
> From there it's just a normal installation like every other module, I presume.

thank, just didn't know what's branches before
